flow-typed/npm/redux-freeze_v0.x.x.js

```diff
@@ -0,0 +1,10 @@
+// flow-typed signature: 48f658870425948aa6aa4b57464b089f
+// flow-typed version: <<STUB>>/redux-freeze_v0/flow_v0.63.1
+
+declare module 'redux-freeze' {
```

Comment:
Probably shouldn't be in this PR?

Reply:
Not exactly 😬 

Doesn't hurt though, #13 already depends on this.